client/js/screens/SignIn/SignIn.js

```diff
@@ -15,136 +19,192 @@ import InputDefaultField from '../../components/InputDefaultField';
+                        <View style={styles.columnCols2}>
+                          <InputDefaultCheckBox
+                            type="checkbox"
+                            name="rememberMe"
+                            rightText="Remeber me"
+                          />
+                          <Text style={styles.forgot}>Forgot Password?</Text>
+                        </View>
 
-            <View style={{...styles.columnCols2, ...styles.wrapSocial}}>
-              <TouchableOpacity style={styles.socialIcon}>
-                <View style={{...styles.btnSocial, ...styles.btnFacebook}}>
-                  <Image
-                    resizeMode="cover"
-                    source={require('../../assets/images/icFacebook.png')}
-                  />
-                  <Text style={{...styles.socialText, ...styles.textFacebook}}>
-                    Facebook
+                        <ButtonDefault
+                          onPress={() => {
+                            if (isErrors) {
+                              Alert.alert(
+                                isErrors,
+                                '',
+                                [
+                                  {
+                                    text: 'OK',
+                                  },
+                                ],
+                                {cancelable: true},
+                              );
+                            } else {
+                              props.handleSubmit();
+                            }
+                          }}
+                          buttonStyle={styles.button}
+                          isActive={!isErrors ? true : false}
+                          title="Sign In"
+                        />
+                      </View>
+                    );
+                  }}
+                </Form>
+                <View>
+                  <Text style={styles.text}>or</Text>
+                  <Text style={styles.text}>
+                    Sign in with your social media account
                   </Text>
-                </View>
-              </TouchableOpacity>
 
-              <TouchableOpacity
-                style={{...styles.socialIcon, ...styles.secondIcon}}>
-                <View style={{...styles.btnSocial, ...styles.btnGoogle}}>
-                  <Image
-                    resizeMode="cover"
-                    source={require('../../assets/images/icGoogle.png')}
-                  />
-                  <Text style={{...styles.socialText, ...styles.textGoogle}}>
-                    Google
+                  <View style={{...styles.columnCols2, ...styles.wrapSocial}}>
+                    <TouchableOpacity style={styles.socialIcon}>
+                      <View
+                        style={{...styles.btnSocial, ...styles.btnFacebook}}>
+                        <Image
+                          resizeMode="cover"
+                          source={require('../../assets/images/icFacebook.png')}
```

Comment:
stylistic thing, but I would put these images at the top (const facebookImage = require(...))